### PR TITLE
Articles Content Layout

### LINKED ISSUE #55: Style Articles Layouts for Politics, Sports, Stocks and Animals

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Created a Content Layout for Politics, Sports, Stocks and Animals components to utilize. 
2. Rearrange the whole layout of the main page. 
3. Added buttons component to the Main Article card. 

## Purpose
Enable users to have a better experience when browsing for some news. 

Closes #55

### CODE REVIEW

**Review by Gjhnsn — APPROVED** [review]
Everything looks good from what I can see, looks like a big redesign! Excited to see how it turns out.

File: the-toad-tribune/src/layout/Animals.tsx [hunk 1]
```diff
@@ -38,32 +36,28 @@ const Animals: React.FC<INewsProps> = ({
...
-        </HeaderContainer>
+        <ImageContainer2>
+          <img src={article?.urlToImage} />
+        </ImageContainer2>
 
-        <ArticleDescriptionContainer>
+        <HeaderContainer2>
+          <h3>{article?.title}</h3>
+          <h4>{article?.author}</h4>
+          <h5>{article?.publishedAt}</h5>
```
Comment (gonzalespaulb) [inline the-toad-tribune/src/layout/Animals.tsx:52]: Very nice use of H tags for the hierarchy of headings.

File: the-toad-tribune/src/layout/Layout.tsx [hunk 2]
```diff
@@ -86,12 +86,12 @@ export default Layout;
 const MainLayoutStyles = styled.div`
   height: 100vh;
   width: 100vw;
-  background-color: green;
+  background-color: hsl(0, 0%, 18%);
   display: grid;
-  grid-row-gap: 0.5em;
-  grid-column-gap: 0.5em;
-  grid-template-columns: repeat(9, 1fr);
+  grid-template-columns: 8% 12% 12% 12% 12% 12% 12% 12% 8%;
```
Comment (gonzalespaulb) [inline the-toad-tribune/src/layout/Layout.tsx:91]: can also do 8% repeat(8, 12%) 8%;

File: the-toad-tribune/src/layout/MainArticle.tsx [hunk 3]
```diff
@@ -60,15 +68,38 @@ const MainArticle: React.FC<INewsProps> = ({
 export default MainArticle;
 
 const MainArticleStyles = styled.div`
-  background-color: blue;
-  grid-area: 2/3/6/9;
-  background-repeat: no-repeat;
-  background-position: center;
+  border: double;
```
Comment (gonzalespaulb) [inline the-toad-tribune/src/layout/MainArticle.tsx:71]: Interesting use of border.

File: the-toad-tribune/src/layout/Layout.tsx (hunk 2 above)
Comment (JayDampitan) [reply the-toad-tribune/src/layout/Layout.tsx:91]: Good to know tol, thanks!

**Review by gonzalespaulb — APPROVED** [review]

File: the-toad-tribune/src/layout/Layout.tsx (hunk 2 above)
Comment (connielion) [reply the-toad-tribune/src/layout/Layout.tsx:91]: * 8% repeat(7, 12%) 8%;

File: the-toad-tribune/src/layout/Layout.tsx [hunk 4]
```diff
@@ -39,7 +39,7 @@ const Layout: React.FC<LayoutProps> = ({
       case "MainLayout":
         return (
           <MainLayoutStyles>
-            {Ads}
+            {/* {Ads} */}
```
Comment (Lap343) [inline the-toad-tribune/src/layout/Layout.tsx:42]: Ads still needed? or you probably going to enable it later

**Review by connielion — APPROVED** [review]
Nice and long PR.

**Review by Lap343 — APPROVED** [review]
Chonky boi but my third eye just opened. Good code

**Pull request merged**